From 1d969f590498a7f66452666a5d3ec08a6d2a7298 Mon Sep 17 00:00:00 2001
From: Kalev Lember <klember@redhat.com>
Date: Wed, 12 Sep 2018 15:02:59 +0200
Subject: [PATCH] dnf: Avoid using deprecated hy_goal_downgrade_to()

Just use hy_goal_install() that does the exact same thing.
---
 backends/dnf/pk-backend-dnf.c | 10 +++-------
 1 file changed, 3 insertions(+), 7 deletions(-)

diff --git a/backends/dnf/pk-backend-dnf.c b/backends/dnf/pk-backend-dnf.c
index 7c8465b73..988dd8fcb 100644
--- a/backends/dnf/pk-backend-dnf.c
+++ b/backends/dnf/pk-backend-dnf.c
@@ -3152,14 +3152,10 @@ pk_backend_install_packages_thread (PkBackendJob *job, GVariant *params, gpointe
 						   "Failed to find %s", package_ids[i]);
 			return;
 		}
-		if (relations[i] == HY_LT) {
-			hy_goal_downgrade_to (job_data->goal, pkg);
-		} else {
-			if (relations[i] == HY_EQ) {
-				dnf_package_set_action (pkg, DNF_STATE_ACTION_REINSTALL);
-			}
-			hy_goal_install (job_data->goal, pkg);
+		if (relations[i] == HY_EQ) {
+			dnf_package_set_action (pkg, DNF_STATE_ACTION_REINSTALL);
 		}
+		hy_goal_install (job_data->goal, pkg);
 	}
 
 	/* run transaction */
-- 
2.19.0

